projects/gen-ai.md

```diff
@@ -0,0 +1,164 @@
+
+The eventual goal is to contribute instrumentation libraries to upstream components. When it's not possible or when libraries need some time to mature,
+these libraries will evolve in contrib repositories.
+
+JS contrib provides component-ownership model with high level of [autonomy](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/CONTRIBUTING.md#component-ownership).
+We can explore if this can be applied to Python.
+
+> [!NOTE]
+> The release cadence of contrib repo needs to support releasing components relatively frequently to
+> patch issues and adapt to upstream changes so that GenAI observability vendors could depend on them.
+
+Python contrib repo is going to support [per-package release](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2875) soon.
+JS contrib release is automated with Release Please - TBD.
+
+## Deliverables
+
+Immediate term:
+- Ship OTel instrumentation libraries for OpenAI (or any other GenAI client) in Python and JS following existing conventions
+
+Middle term:
+- Ship OpenTelemetry (or native) instrumentations for popular GenAI client libraries in Python and JS covering chat calls
+- Evolve GenAI semantic conventions to cover other popular GenAI operations such as embeddings, image or audio generation
+
+As a result we should have feature parity with the instrumentations of existing GenAI Observability vendors for a set of client
+instrumentation libraries that all vendors can depend upon.
+
+Long term:
+- Implement instrumentations for GenAI orchestrators and GenAI frameworks for popular libraries in different languages
+- Evolve GenAI and other relevant conventions (DB) to cover complex multi-step scenarios such as RAG
+- Propose mature instrumentations to upstream libraries/frameworks
+
+## Staffing / Help Wanted
+
+The development will start in the OTel contrib repositories and/or in upstream libraries following ownership model that exist in this repo.
+
+Some of the repositories (such as opentelemetry-js-contrib) provide a lot of autonomy to the [component owners](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/CONTRIBUTING.md#component-ownership).
+
+**Project Leads:**
+
+- @drewby
+- @nirga
+
+**Sponsoring Members:**
+
+- @lmolkova
+- TBD
+
+**Engineers:**
+
+| Alias | Language | Company/Project | Time commitment |
+| ----- | -------- | ----------------|---------------- |
+| @AmanAgarwal041 | JS & Python | OpenLIT | |
+| @alizenhom  | JS & Python | Langtrace | |
+| @codefromthecrypt | JS & Python | Elastic | |
+| @drewby  | JS & Python | Microsoft | |
+| @galkleinman | JS & Python | Traceloop | |
+| @gyliu513 | JS & Python | IBM | |
+| @karthikscale3 | JS & Python | Langtrace | |
+| @lmolkova | Python | Microsoft | | 16 hours/week |
+| @lzchen | Python | Microsoft |  | |
+| @nirga  | JS & Python | Traceloop | |
+| @patcher9 | JS & Python | OpenLIT | |
+| @trentm | JS | Elastic | |
+| @xrmx | Python | Elastic | |
```

Comment:
I've been absent lately due to an incredibly full month of public events. In many cases, the hours would count I guess, as I met and spoke to folks about genai specs, and in some external projects and working on having them adopt them. Anyway, 4hrs a week seems something that makes sense esp on average.

```suggestion
| @xrmx | Python | Elastic | |
| @codefromthecrypt | JS & Python | Elastic | 4 hours/week |
```

Reply:
ps I also have done work in dotnet, java and go, just limiting langs to the ones we have in scope.